content/Building Engineering Goals.md

```diff
@@ -0,0 +1,66 @@
+# Building engineering goals
+
+## Our vision
+Rootstrap is a digital agency that focuses on web and app development. We’re based in Uruguay and the United States. As a digital agency, our vision is to become globally recognized digital masters. This epithet doesn’t just sound nice. It also means a lot: have one of the most skilled full teams, have people who love where they work, build products our client and their users' love, and much more.
+
```

Comment:
Shouldn't it say `have one of the most skillful teams`? Also `'` should be deleted after `users`

Reply:
@CamilaMoscatelli seems like `skilled` is more appropriate here. 
```
In sum, skilled means having lots of training/experience and it is the best word to use when talking about workers and laborers. Skillful means expert at a particular job, but not necessarily because of training or experience.
```
http://www.learnersdictionary.com/qa/the-difference-between-skilled-and-skillful

@juanlacruz just to let you know, a copywriter took a look at grammar, in this case, I think the current and your solution are both corrects.